core/src/main/scala/org/apache/spark/internal/config/package.scala

```diff
@@ -715,6 +716,26 @@ package object config {
       .checkValue(v => v > 0, "The max failures should be a positive value.")
       .createWithDefault(40)
 
+  private[spark] val UNSAFE_EXCEPTION_ON_MEMORY_LEAK =
+    ConfigBuilder("spark.unsafe.exceptionOnMemoryLeak")
+      .booleanConf
+      .createWithDefault(false)
+
+  private[spark] val UNSAFE_OFFHEAP =
+    ConfigBuilder("spark.unsafe.offHeap")
```

Comment:
I think this is wrong: this is just the old name of `spark.memory.offHeap.enabled`, so creating a new config here is wrong.

Reply:
`spark.unsafe.offHeap` is deprecated long time ago, I think we can get rid of it.

Reply:
I see, done